src/commands/svc/enable.ts

```diff
@@ -0,0 +1,32 @@
+import { CliUx, Command } from '@oclif/core'
+import { BundleService } from '../../services/bundle-service'
+import { SvcService } from '../../services/svc-service'
+
+export default class Enable extends Command {
+  static description =
+    'Enables an available docker services found in services folder'
```

Comment:
Also change topic description to "Manage auxiliary services"

Reply:
done. added in `package.json`